cea/default.config

```diff
@@ -790,6 +790,19 @@ substation-heating-systems.type = ListParameter
 substation-heating-systems.help = List of cooling loads to be supplied if we are in DC mode
 substation-heating-systems.category = Advanced
 
+temperature-control = VT
+temperature-control.type = ChoiceParameter
+temperature-control.choices = VT, CT
+temperature-control.help = Type of control strategy for plant operation, either VT (Variable temperature, variable flow) or CT (Constant Temperature, variable flow)
+temperature-control.category = Advanced
+
+
+plant-supply-temperature = 80
+plant-supply-temperature.type = RealParameter
+plant-supply-temperature.help = Supply temperature at plants if temperature-control is set to 'CT'
```

Comment:
maybe mention the unit here? K? C? yes. i just outed myself as too stupid to gues... [C]?

Reply:
I added the unit (it was C)